setup.py

```diff
@@ -20,7 +20,7 @@
 extras_require = {'testing': ['pytest', 'pytest-flake8']}
 
 PACKAGE_NAME = 'shadems'
-__version__ = '0.5.1.1'
+__version__ = '0.5.2'
```

Comment:
I don't think the version should be changed here. It should be bumped as part of a release process, rather than bundled into a fix for other concerns.

Reply:
ok will revert this